### ISSUE TITLE
Add headers to messages

### ISSUE THREAD

**Issue by levik-buka** [CommunityHiQ/Frends.Community.RabbitMQ#19]
Hi,
Would it be ok, if I will extend Write and Read task parameters with header information same way it is done in Frends.Web library?
```
public class Header
{
    public string Name { get; set; }
    public string Value { get; set; }
}

public Header[] Headers { get; set; }
```

I need some easy way to recognize the content of the queue message and RabbitMQ offers good solution for it:
https://www.rabbitmq.com/tutorials/amqp-concepts.html#messages

Br,
Lev

**Comment by OssiGalkin** [comment]
Hi!

That sounds good. 

If possible add new parameters at the end. This way frends can automatically update tasks. Of course it is not always possible e.g. it could lead very award UI.

Br,
Ossi

**Comment by OssiGalkin** [comment]
Fixed in pr #21 (or version 1.6.0).
